### ISSUE TITLE
Difficulty dragging boards on some mobiles

### ISSUE THREAD

**Issue by bovas85** [davestewart/app-diary#1]
I can't move boards left to right, only right to left.

This is when opening the demo link

**Comment by davestewart** [comment]
Hmm. I can't reproduce that!

![](http://g.recordit.co/zg3rzEeQOu.gif)

Is this on a particular OS/Browser?

EDIT 1: Ah - just seen the word "mobile" in the title. Can you provide more info?
EDIT 2: Just tried on Pixel 2 / Chrome and it works just like the desktop

I use this lib for the DnD: https://github.com/kutlugsahin/vue-smooth-dnd

**Comment by bovas85** [comment]
I have a OnePlus 5 with latest chrome.

I'm trying to drag the first board to the right, which works, but the anchor point threshold is perhaps too far and it comes back to the starting point.

**Comment by davestewart** [comment]
Hmm. That's going to be difficult for me to reproduce! 😢 

Could be a great opportunity for your first PR! Either here, or I suspect in the library I mentioned above.

**Comment by davestewart** [comment]
So the author of Smooth DnD has a demo page. 

- https://kutlugsahin.github.io/vue-smooth-dnd/

Want to try that and see how your phone copes?

**Comment by bovas85** [comment]
Yeah I think the actual lib is broken on mobile, at least at my size and phone, which is quite common.

Does it work for you and your phone when dragging stuff?

**Comment by bovas85** [comment]
maybe worth looking at other options?
https://github.com/vuejs/awesome-vue#drag-and-drop

**Comment by davestewart** [comment]
> Does it work for you and your phone when dragging stuff?

Yeah, no problems at all!

> maybe worth looking at other options?

If you check the diary, you'll see I tried a bunch of them, and actually Vue Smooth DnD was the one I ended up at:

- https://davestewart.github.io/app-diary/#/diary/v0.1.0?id=_30m-views-and-components

I actually love how it works, but it's unfortunate it doesn't work on all phones.

Given that this is open source, and all the interface and store work has already been done, do you fancy looking at it!?

After all, the project exists online as a learning exercise; for me - rapid prototyping, for others - whatever suits them!

**Comment by bovas85** [comment]
Yes I will clone and contribute. There's surely a threshold somewhere that we can fine tune

**Comment by davestewart** [comment]
> Yes I will clone and contribute.

Though saying that, it could be useful to create an issue on the Vue Smooth DnD repo first.

If it's a known issue, that saves you some work. If it's an unknown issue, maybe they can fix it?

**Comment by bovas85** [comment]
yes that's the intention, they have the bug on their demo itself.

You can actually check yourself by inspecting element, simulating mobile size (I tried around 320px) and trying to drag the whole board to the right.

**Comment by davestewart** [comment]
Oh yeah, that's weird. Every time I try to drag the screen pans!
